exporter/elasticsearchexporter/internal/serializer/otelserializer/serializeprofiles/transform.go

```diff
@@ -18,6 +18,8 @@ import (
 	semconv "go.opentelemetry.io/otel/semconv/v1.25.0"
 )
 
+var Now = time.Now
```

Comment:
I know testing with time is hard, but adding a global Now function that tests manipulate is not ideal. Stuff like this will break and race when tests run in parallel. Is it possible to write a custom comparison function to either strip / ignore time fields instead?

Reply:
:+1: Exactly what concerned me as well. A checkAndReset() function for timestamps make much more sense.